src/frontend/screens/WineManager/index.tsx

```diff
@@ -55,17 +55,22 @@ export default function WineManager(): JSX.Element | null {
     value: 'gpt',
     enabled: !isLinux
   }
+  const wineCrossover: WineManagerUISettings = {
+    type: 'Wine-Crossover',
+    value: 'winecrossover',
+    enabled: !isLinux
+  }
 
   const [repository, setRepository] = useState<WineManagerUISettings>(
-    isLinux ? protonge : gamePortingToolkit
+    isLinux ? protonge : isIntelMac ? wineCrossover : gamePortingToolkit
```

Comment:
I think those double ternary are not clean, maybe a method for that would be better.